lib/picker.js

```diff
@@ -629,10 +643,11 @@ function PickerConstructor( ELEMENT, NAME, COMPONENT, OPTIONS ) {
             ).
 
             // On focus/click, open the picker.
-            on( 'focus.' + STATE.id + ' click.' + STATE.id, function(event) {
+            on( 'focus.' + STATE.id + ' click.' + STATE.id,
+            debounce(function(event) {
                 event.preventDefault()
                 P.open()
-            })
+            }, 300))
```

Comment:
300 seems rather long..if it's a race condition introduced in Chrome 73, I can't imagine it would need that long 🤔

Reply:
We should also add a comment here as to why it's debounced (with a reference to the issue).

Reply:
Ok, which value would you recommend? 100ms or lower?

Reply:
I would assume that since it's a race condition at the browser level, we wouldn't need anything more than a few milliseconds. To be safe, I think just 50ms should be sufficient. At 100ms, it starts to impact the experience minutely.